corehq/apps/cloudcare/views.py

```diff
@@ -187,6 +189,9 @@ def get_option_apps(self, request, domain):
         apps = self.get_web_apps_available_to_user(domain, restore_as)
         return JsonResponse(apps, safe=False)
 
+    def _get_domain_obj(self, domain):
```

Comment:
Minor: you don't really gain anything by making this a function, given that it's only called once and it's a one-liner.

Reply:
done